website/docs/releases/2024/v2024.2.md

```diff
@@ -0,0 +1,10180 @@
+---
+title: Release 2024.2 (unreleased)
+slug: /releases/2024.2
+---
+
+:::::note
+2024.2 has not been released yet! We're publishing these release notes as a preview of what's to come, and for our awesome beta testers trying out release candidates.
+:::::
+
+## Highlights
+
+-   **Remote Access Control** <span class="badge badge--primary">Enterprise</span> Access machines over RDP, SSH and VNC from authentik
+
+-   **Audit log** <span class="badge badge--primary">Enterprise</span> See what fields were changed when objects are updated
+
+-   **Session location and network binding** Increase security by preventing session theft
+
+-   **Wizard to simplify creating applications and providers**
+
+## Breaking changes
+
+### Manual action is required
+
+-   **Tenants have been renamed to brands**
+
+    Tenants, which used to allow the changing of branding configuration and default flows and a couple other settings depending on which domain authentik was accessed from, have been renamed to _brands_. This is due to this release including full tenancy (see [here](#new-features))
+
+    Existing _tenant_ objects will automatically be renamed to _brand_ objects. The API endpoints associated with _brands_ have also been renamed.
+
+    Blueprints using `authentik_tenants.tenant` will need to be changed to use `authentik_brands.brand`.
+
+    See documentation for _brands_ [here](../../core/brands.md)
+
+    Also, **the event retention settings configured in brands (previously tenants, see above) has been removed and is now a system setting**, managed in the admin interface or via the API (see above).
+
+    **There is no built-in migration path for this change.** If you set something other than the default (`days=365`), you will need to update the setting in the admin interface.
+
+-   **Helm chart breaking changes**
+
+    The Helm Chart has a number of breaking changes. Find out more in the [chart release notes](https://github.com/goauthentik/helm/releases/tag/authentik-2024.2.0-rc1).
+
+### Manual action may be required
+
+-   **Required `offline_access` scope for Refresh tokens**
+
+    The OAuth2 provider ships with a new default scope called `offline_access`, which must be requested by applications that need a refresh token. Previously, authentik would always issue a refresh token for the _Authorization code_ and _Device code_ OAuth grants.
+
+    Applications that require will need their configuration update to include the `offline_access` scope mapping.
```

Comment:
Applications that require __what exactly?___ ....

Reply:
A refresh token, apparently I missed a word x)